spec/requests/users_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /users' do
+    it 'it returns success http status code' do
+      get '/users'
+      expect(response.status).to eq(200)
+    end
+
+    it 'returns the correct template' do
+      get '/users'
+      expect(response).to render_template('users/index')
+    end
+
+    it 'returns body that include placeholder text' do
+      get '/users'
+      expect(response.body).to include('All Users')
+    end
+  end
+end
```

Comment:
As stated earlier, you have only written tests for the `index` action and you haven't written for the `show` action😉. kindly do well to write some tests for the show action as this is a requirement in this milestone👍🏼.

<img width="816" alt="image" src="https://github.com/toyman640/blog/assets/64643995/105508e7-d68f-4707-a952-8e77d38895df">